src/database/queries/deleteCohortStudent.js

```diff
@@ -0,0 +1,15 @@
+const dbConnection = require('../db_connection');
+
+const deleteStudentQuery = (deleteStdData, cb) => {
+  const { stdId, cohortID } = deleteStdData;
```

Comment:
You could also say:
```js
const deleteStudentQuery = ({ stdId, cohortID }, cb) => {
```
But yours work too, so no need to change it.

Reply:
thanks for this idea , I did it